Mathlib/Order/Heyting/Basic.lean

```diff
@@ -142,7 +142,7 @@ class GeneralizedCoheytingAlgebra (α : Type*) extends Lattice α, OrderBot α,
 /-- A Heyting algebra is a bounded lattice with an additional binary operation `⇨` called Heyting
 implication such that `a ⇨` is right adjoint to `a ⊓`. -/
```

Comment:
Could you please use the proper Lean 4 notation here?
```suggestion
implication such that `(a ⇨ ·)` is right adjoint to `(a ⊓ ·)`. -/
```

Reply:
Same modification is needed here?

```lean
class GeneralizedHeytingAlgebra (α : Type*) extends Lattice α, OrderTop α, HImp α where
  /-- `a ⇨` is right adjoint to `a ⊓` -/
  le_himp_iff (a b c : α) : a ≤ b ⇨ c ↔ a ⊓ b ≤ c
```

Reply:
Yes indeed

Reply:
done!